### ISSUE TITLE
Request: optional non-whitelisted mode

### ISSUE THREAD

**Issue by junkyardsparkle** [Seva-coder/Finder#14]
Thanks for the work on this app. I have previously had FindMyPhone installed and it proved useful a couple of times, but my experience tells me that I really don't know in advance what numbers should be whitelisted. Are you opposed to offering a mode which only relies on the message contents for activation?

I've tried enabling the "remote adding of new phone numbers", but haven't been able to get it to work in limited testing so far (I'll try again when I can). Adding this extra step, even if working correctly, seems like it might be a hardship under some circumstances, such as if you need the assistance of somebody who isn't a close friend; in this case you would be trying to take as little of a person's time as possible.

**Comment by Seva-coder** [comment]
I agree, adding such a mode is not too difficult. It may be useful in some cases.
Have you checked the basic functionality? (with whitelist). May be problem with stopping app by system

**Comment by junkyardsparkle** [comment]
Great, thanks! Yes, I think it's related to system power-saving settings, but I haven't had a chance to troubleshoot it any more yet. I'll be replacing the phone soon, and will be more interested in getting this working on the new one.
